druid/examples/wasm/build.rs

```diff
@@ -61,14 +74,22 @@ mod examples {
                 example_str.to_string()
             };
 
+            // Add an entry to the index.html file.
+            let index_entry = format!(
+                "<li><a href=\"./html/{name}.html\">{name}</a></li>",
+                name = js_entry_fn_name
```

Comment:
The html file is named after the example name
```suggestion
                name = example_str
```